### PR TITLE
Support for ItemizedPaymentInformation element in ShipmentConfirmRequest

### PR DESCRIPTION
**Pull request by thijsw**
Hi,

I've added support for the `<ItemizedPaymentInformation>` element in a Shipment Confirm Request per the UPS documentation. I've tried to follow the current coding style of the code base.

### CODE REVIEW

File: src/Shipping.php [hunk 1]
```diff
@@ -305,9 +282,51 @@ private function createConfirmRequest(
             } elseif ($shipment->getPaymentInformation()->getConsigneeBilled()) {
                 $paymentNode->appendChild($xml->createElement('ConsigneeBilled'));
             }
-//         TODO: create ItemizedPaymentInformation class and required subclasses for node processing
-//        } elseif ($shipment->getItemizedPaymentInformation()) {
-//            $paymentNode = $shipmentNode->appendChild($xml->createElement('ItemizedPaymentInformation'));
+        } elseif ($shipment->getItemizedPaymentInformation()) {
```
Comment (stefandoorn) [inline src/Shipping.php]: Can we also move this towards using toNode?

File: src/Entity/BillShipper.php [hunk 2]
```diff
@@ -18,6 +18,11 @@ class BillShipper
     private $creditCard;
 
     /**
+     * TODO: AlternatePaymentMethod?
```
Comment (stefandoorn) [inline src/Entity/BillShipper.php]: Is this work in progress?

File: src/Shipping.php (hunk 1 above)
Comment (thijsw) [inline src/Shipping.php]: done

File: src/Entity/BillShipper.php (hunk 2 above)
Comment (thijsw) [inline src/Entity/BillShipper.php]: not anymore :)

File: src/Entity/BillThirdParty.php [hunk 3]
```diff
@@ -63,4 +67,32 @@ public function setAccountNumber($accountNumber)
...
+    public function toNode(DOMDocument $document = null)
+    {
+        if (null === $document) {
+            $document = new DOMDocument();
+        }
+
+        $node = $document->createElement('BillThirdParty');
+
+        $btpNode = $node->appendChild($document->createElement('BillThirdPartyShipper'));
+        $btpNode->appendChild($document->createElement('AccountNumber', $this->getAccountNumber()));
```
Comment (stefandoorn) [inline src/Entity/BillThirdParty.php]: Can we offload in here also to custom entities? So a 'BillThirdPartyShipper' with it's own `toNode` method? It feels more like following the existing flow than creating sub entities inside this class.

**Comment by scottcwilson** [comment]
Argh I didn't realize there was an outstanding PR for this stuff.  I just submitted my own PR to do it.

**Comment by thijsw** [comment]
@scottcwilson  Ahh too bad, and it's not the most fun work to do either... I must say I was a bit sloppy in not fixing the addressed issues in this PR otherwise I bet it'd be merged a lot sooner.

**Comment by stefandoorn** [comment]
Hm, in the meanwhile we have 3 PR's actually on this subject. One by @thijsw, another one by @stephenjwinn and a recent one of @scottcwilson. Could you maybe team up in order to implement it? :-) I'm not using UPS actively right now and having time constraints, but can do reviewing and merging.

**Comment by scottcwilson** [comment]
Please let me know if this PR should be closed @thijsw

**Comment by stephenjwinn** [comment]
You can close mine

**Comment by scottcwilson** [comment]
Thank you @stephenjwinn.  Done.

**Comment by gabrielbull** [comment]
@thijsw This library has no maintainer and is looking for one.

**Pull request closed without merging**